content/articles/visualizing-repositories-using-pygal/index.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /visualizing-repositories-using-pygal/
+title: Visualizing repositories using Pygal
+description: In this article, you'll learn how to construct a visualization that shows the comparative popularity of Python works on GitHub using Pygal.
+author: duncan-ndegwa
+date: 2021-09-03T00:00:00-04:35
+topics: [API]
+excerpt_separator: <!--more-->
+images:
+
+ - url: engineering-education\content\articles\visualizing-repositories-using-pygal\hero.jpg
+   alt: Visualizing repositories using Pygal hero image
+---
+
+### Introduction
+Pygal allows us to create a range of graphs and charts. In this article, you'll learn how to construct a visualization that shows the comparative Python's prominence works on github.com. We'll create an interactive bar chart, where the height of each bar will represent the sum of stars earned by the project. By clicking a bar, you'll be sent to the project's GitHub page.
+<!--more-->
+
+### Prerequisites
+You'll need some Python skills to follow along with this tutorial. You must be in a position to work with web APIs in Python. Also, you need to have [PIP](https://pip.pypa.io/en/stable/installation/) installed.
+
+### Installing pygal
+To install Pygal, run the command below;
+
+```python
+pip install pygal
+```
+
+You can visit [Pygal's official site](http://www.pygal.org/en/stable/installing.html) for more information on the installation of Pygal.
+
+### The pygal gallery
+The Pygal gallery is a collection of charts that Pygal can generate.
+
+Check the chart types page to explore what types of charts Pygal can generate: Select **Documentation**, finally **Chart types** in [pygal](http://www.pygal.org/). You can examine how the visualizations are created by looking at the source code for each case.
```

Comment:
```suggestion
Check the chart types page to explore what types of charts Pygal can generate: Select **Documentation**, finally **Chart types** in [pygal](http://www.pygal.org/). You can examine how the visualizations are created by looking at the source code for each case.
```
You can also reorder this statement as this ....
"To grasp more of the chart types that can be generated by Pygal, head over to the documentation tab on the chart type page. (issue the link).
Additionally, you can examine how the visualizations are created by exploring the source code for each.